doc/user/content/transform-data/json.md

```diff
@@ -0,0 +1,227 @@
+---
+title: "JSON to SQL"
+description: "Create a view from sample JSON data."
+menu:
+  main:
+    parent: 'transform'
+    name: JSON
+    weight: 20
+---
+
+Working with JSON formatted data in SQL can be tedious, because you need to constantly deconstruct
+the JSON blob. The below tool accepts a JSON sample and will generate a SQL view with the
+individual fields mapped to columns.
+
+
+<div class="json_widget">
+    <span class="input_container">
+        <span class="input_container-text">
+            <input id="view_name" placeholder="View Name">
+            <input id="source_name" placeholder="Relation Name">
+            <input id="column_name" placeholder="JSON Column Name">
+        </span>
+    <fieldset class="input_container-radio">
+        <legend>Kind of SQL Object</legend>
+        <span>
+            <input type="radio" id="view" name="type_view" value="view"/>
+            <label for="view">View</label>
```

Comment:
UX wise, it's probably a good idea to default this to selected.

Reply:
added!